File: RAPTOR Projects/RandomNumber/RandomNumber.cpp
#include <iostream>
#include <string>

using namespace std;
int main()
{
   ?? randnum2;
   ?? maxnum3;
   ?? randnum;
   ?? randnum3;
   ?? bignumber;
   ?? randnum4;
   ?? maxnum2;
   ?? minnum;
   ?? maxnum;

   bignumber =(9*9) ** 9;
   minnum =0;
   maxnum =bignumber;
   maxnum2 =600;
   maxnum3 =10;
   open_graph_window(600,600);
   while (1)
   {
      randnum =random_generator.NextDouble()*(maxnum-minnum)+1+minnum;
      randnum3 =random_generator.NextDouble()*(maxnum2-minnum)+1+minnum;
      randnum4 =random_generator.NextDouble()*(maxnum3-minnum)+1+minnum;
      randnum2 =floor(randnum);
      cout << "Your random number is: "+randnum2 << endl;      set_window_title("TITLE"+randnum2);
      randnum3 =floor(randnum3);
      randnum4 =floor(randnum4);
      draw_circle(randnum3,randnum3*randnum4,randnum4,red,true);
      draw_circle(randnum3*randnum4,randnum3,randnum4,red,true);
      if (randnum2>=bignumber)) break;
   }

   return 0;
}

File: RAPTOR Projects/RandomNumber/RandomNumber.cs
using System;
using System.IO;
using System.Text;

namespace RandomNumber
{
   public class main_class
   {
      static System.Random random_generator = new System.Random();
      public static void Main(string[] args)
      {
         ?? randnum2;
         ?? maxnum3;
         ?? randnum;
         ?? randnum3;
         ?? bignumber;
         ?? randnum4;
         ?? maxnum2;
         ?? minnum;
         ?? maxnum;
      
         bigNumber =(9*9) ** 9;
         minNum =0;
         maxNum =bigNumber;
         maxNum2 =600;
         maxNum3 =10;
         open_graph_window(600,600);
         while (1)
         {
            randNum =random_generator.NextDouble()*(maxNum-minNum)+1+minNum;
            randNum3 =random_generator.NextDouble()*(maxNum2-minNum)+1+minNum;
            randNum4 =random_generator.NextDouble()*(maxNum3-minNum)+1+minNum;
            randNum2 =floor(randNum);
            Console.WriteLine("Your random number is: "+randNum2);
            set_window_title("TITLE"+randNum2);
            randNum3 =floor(randNum3);
            randNum4 =floor(randNum4);
            draw_circle(randNum3,randNum3*randNum4,randNum4,red,true);
            draw_circle(randNum3*randNum4,randNum3,randNum4,red,true);
            if (randNum2>=bigNumber)) break;
         }
      }
   }
}
